solidity/dashboard/src/pages/rewards/TBTCRewardsPage.jsx

```diff
@@ -8,15 +8,18 @@ import ProgressBar from "../../components/ProgressBar"
 import Timer from "../../components/Timer"
 import TBTCRewardsDataTable from "../../components/TBTCRewardsDataTable"
 import { colors } from "../../constants/colors"
+import { ECDSARewardsHelper } from "../../utils/rewardsHelper";
 
 const TBTCRewardsPage = () => {
+  const remainingTime = ECDSARewardsHelper.remainingTimeToFinishCurrentInterval()
```

Comment:
I think we can easier. `const currentIntervalEndOf = ECDSARewardsHelper.intervalEndOf(ECDSARewardsHelper.currentInterval)` and then `<Timer target={currentIntervalEndOf} />`